### ISSUE TITLE
recyclerView中  apng会莫名的自动停止播放（偶尔会出现），刷新一下就又好了

### ISSUE THREAD

**Issue by ThinkAgains** [penfeizhou/APNG4Android#178]
### New Issue Checklist

- [ ] I have searched for a similar issue in the [project](https://github.com/penfeizhou/APNG4Android/issues) and found none

### Issue Info

| Info                         | Value                                                 |      |
| ---------------------------- | ----------------------------------------------------- | ---- |
| Device Info                  | e.g. Huawei Mate 30 pro                               |      |
| System Version               | e.g.  6.0                                             |      |
| APNG4Android Library Version | e.g.  1.0.0                                           |      |
| Repro rate                   | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo project  | e.g. does it happen with our demo project?            |      |
| Demo project link            | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

所有设备

**Comment by ThinkAgains** [comment]
glide直接加载会出现这种问题，加载本地没这个问题

**Comment by penfeizhou** [comment]
请给出具体的Demo代码

**Comment by penfeizhou** [comment]
有问题请开新issue.
